ads/google/doubleclick.js

```diff
@@ -283,8 +283,7 @@ export function selectGptExperiment(data) {
 export function writeAdScript(global, data, gptFilename) {
   const url =
   `https://www.googletagservices.com/tag/js/${gptFilename || 'gpt.js'}`;
-  if (gptFilename || data.useSameDomainRenderingUntilDeprecated != undefined
```

Comment:
We still get quite a few ad requests from the GLADE_OPT_OUT branch, unfortunately we cannot tell for what reason very easily due to all of these being grouped together as opt out. How do we know that this is safe to just turn off?

1) What if we split these out (or at least the explicit opt-out), so that we can verify the actual amount of traffic from it?
and/or
2) What if rather than flipping it off 100% which is slow to revert in an emergency we setup an experiment which ignores this some fraction of the time. We can gradually increase it as a way to urge pubs to change without breaking all of their ads at once. This also gives us a faster mechanism to change things if anything goes wrong.